lib/tool_belt/commands/check_deprecation_warnings.rb

```diff
@@ -0,0 +1,68 @@
+require 'parser/current'
+require_relative '../deprecation_warning_parser.rb'
+
+module ToolBelt
+  module Command
+    class CheckDeprecationWarningsCommand < Clamp::Command
+
+      parameter "repo_path", "Path (relative to tool_belt) of repository to check"
+      option "--version", "VERSION", "Release version to check for", required: true
+      option "--paths", "PATHS", 
+             "Patterns within the repository to search, separated by a comma.",
+             default: "app/**/*.rb"
+
+      def execute
+        root_path = File.expand_path("../../../../#{repo_path}", __FILE__)
+        globs_to_scan = paths.split(",").map { |path| File.expand_path(path, root_path) }
+        puts "\nScanning for deprecation warnings in #{globs_to_scan.join(", ")}\n\n"
```

Comment:
I'm not sure there's a scenario where I would be checking multiple directories for the same version so maybe a single path is enough. Can you think of one?